src/views/Overview/Overview.vue

```diff
@@ -8,25 +8,25 @@
             <b-col sm="6">
               <dl>
                 <dt>MODEL</dt>
-                <dd>{{ system.Model || 'N/A' }}</dd>
+                <dd>{{ serverInfo.Model || '--' }}</dd>
```

Comment:
How do you feel about separating each serverInfo property into its own state in the OverviewStore? That way we can consistently handle the undefined/default values in the store instead of the template.
So instead of `serverInfo.Model` ->  `serverModel`